back-end/public/stylesheets/style.css

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
Stylesheets coming from the generator won't be needed as create-react-app will be generating it's own CSS - this folder and its files can be removed

Reply:
Updated! Yes, I assumed we would likely remove this later, but no time like the present, I guess.